### PR TITLE
Create Fact “ground-nesting-bees”

### PR DESCRIPTION
**Pull request by NicolasRobertson**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/facts/bee-nesting-underground.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  src: ../images/beenestingunderground.jpg
+  license: CC by 2.0
+  url: https://www.flickr.com/photos/minicooper93402/6828563216
+  creator: devra
+source:
+  name: Wild Bee Nests and Building Wild Bee Houses Nesting Habitat
+  url: https://www.beelab.umn.edu/wild-bees/wild-bees-and-houses
+category: Habitat
+---
+30-40% of bees find cavities to live in, such as hollow branches/stems or dead trees. Inside, they use mud to create chambers for their eggs. Leave fallen trees/branches around your yard to attract bees.
```
Comment (floogulinc) [inline src/facts/bee-nesting-underground.md]: I think you copied the wrong text here.
```suggestion
60-70% of bees live underground. They like to dig tunnels in sandy soil with nothing growing in it. Attract bees using bare, undisturbed soil around your yard.
```

File: src/facts/bee-nesting-underground.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Bee Nesting - Underground
```
Comment (floogulinc) [inline src/facts/bee-nesting-underground.md]: I think we should rename to "Ground Nesting Bees"

File: src/facts/bee-nesting-underground.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+image:
+  alt: Bee Nesting - Underground
+  src: ../images/beenestingunderground.jpg
+  license: CC by 2.0
```
Comment (floogulinc) [inline src/facts/bee-nesting-underground.md]:
```suggestion
  license: CC BY 2.0
```

File: src/facts/bee-nesting-underground.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  url: https://www.flickr.com/photos/minicooper93402/6828563216
+  creator: devra
+source:
+  name: Wild Bee Nests and Building Wild Bee Houses Nesting Habitat
+  url: https://www.beelab.umn.edu/wild-bees/wild-bees-and-houses
+category: Habitat
```
Comment (floogulinc) [inline src/facts/bee-nesting-underground.md]:
```suggestion
category: habitat
```

File: src/facts/bee-nesting-underground.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+title: Bee Nesting - Underground
+image:
+  alt: Bee Nesting - Underground
+  src: ../images/beenestingunderground.jpg
+  license: CC by 2.0
+  url: https://www.flickr.com/photos/minicooper93402/6828563216
+  creator: devra
+source:
+  name: Wild Bee Nests and Building Wild Bee Houses Nesting Habitat
```
Comment (floogulinc) [inline src/facts/bee-nesting-underground.md]:
```suggestion
  name: Wild bee nests
```

**Review by floogulinc — CHANGES_REQUESTED** [review]

File: src/facts/bee-nesting-underground.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+title: Ground Nesting Bees
+image:
+  alt: Ground Nesting Bees
```
Comment (floogulinc) [inline src/facts/bee-nesting-underground.md]: The alt text should describe the image rather than just repeat the fact title.

**Review by floogulinc — CHANGES_REQUESTED** [review]
Please rename the file to `ground-nesting-bees.md`

**Review by floogulinc — CHANGES_REQUESTED** [review]
Please rename the file to `ground-nesting-bees.md`

File: src/facts/bee-nesting-underground.md (hunk 6 above)
Comment (kklamberty) [inline src/facts/bee-nesting-underground.md]:
```suggestion
  alt: Sandy, undisturbed soil with burrows
```

**Review by kklamberty — COMMENTED** [review]
I made a suggestion about an alt tag for the image that does not just repeat the title (after looking at the image). You can just commit that suggestion if you think it captures the image well.

**Review by kklamberty — APPROVED** [review]
I need to apologize here - I looked back at an outdated request for changes from @floogulinc and you had already changed the alt. You can feel free to use my suggestion or to leave the change as you made it before. I was looking at the wrong version of the world.

**Review by floogulinc — APPROVED** [review]

**Pull request merged**